src/commands/Auth/authCommand.ts

```diff
@@ -6,7 +6,7 @@ import { prompt } from 'inquirer';
 import { isDomain } from '../../utils/string';
 
 const authCommand = (program: CommanderStatic) => {
-  program
+  return program
```

Comment:
I added a `return` statement here to call the `program.opts()` to get the option values that are parsed into the command.
Ex: https://github.com/kintone-labs/kintone-cli/pull/106/files#diff-7ebe0f9dced1b1f6a9cda1b0a8977ac50be5743b25a3f8863aa6c2aac5436d0dR52